recipes/r-bigsnpr/meta.yaml

```
@@ -0,0 +1,152 @@
+{% set version = '1.12.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bigsnpr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bigsnpr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bigsnpr/bigsnpr_{{ version }}.tar.gz
+  sha256: 443a5529ae2596eb3cf8f19760893b96beba4f12a4decbf6bea645db4d2187f6
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  # This is needed to suppress a warning about R.dll not being found
+  missing_dso_whitelist:
+    - '*/R.dll'            # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - libgomp                      # [linux]
+    - llvm-openmp                  # [osx]
+  host:
+    - r-base
+    - r-dbplyr
+    - r-dplyr
+    - r-fdrtool
+    - r-matrix >=1.3.0
+    - r-rcpp
+    - r-rcpparmadillo >=0.9.600
+    - r-bigassertr >=0.1.6
+    - r-bigparallelr
+    - r-bigreadr
+    - r-bigsparser >=0.6
+    - r-bigstatsr >=1.5.11
+    - r-bigutilsr >=0.3.3
+    - r-data.table >=1.12.4
+    - r-dorng
+    - r-foreach
+    - r-ggplot2
+    - r-magrittr
+    - r-quadprog
+    - r-r.utils
+    - r-rmio
+    - r-rmutil 
+    - r-roptim >=0.1.6
+    - r-rsqlite
+    - r-runonce >=0.2.3
+    - r-vctrs
+    - r-xgboost
```

Move to `test` only:

```suggestion
    - r-rmio
    - r-roptim >=0.1.6
    - r-runonce >=0.2.3
    - r-vctrs
```